backend/postgres/postgres.go

```diff
@@ -874,6 +874,67 @@ func (be *postgresBackend) GetOrchestrationRuntimeState(ctx context.Context, wi
 	return state, nil
 }
 
+func (be *postgresBackend) GetInstanceHistory(ctx context.Context, wi *backend.GetInstanceHistoryRequest) (*backend.GetInstanceHistoryResponse, error) {
+	if err := be.ensureDB(); err != nil {
+		return nil, err
+	}
+
+	rows, err := be.db.Query(
+		ctx,
+		"SELECT EventPayload FROM History WHERE InstanceID = $1 ORDER BY SequenceNumber ASC",
```

Comment:
I think we are missing the pgSize and continuationToken here and for the list()... unless I'm missing it?

Reply:
This backend implementation is only used for durabletask CI testing, so I don't think it matters too much. We have more robust testing on the Dapr side which is what will be used.